.github/workflows/hosted_inference_e2e_test_production.yml

```diff
@@ -0,0 +1,30 @@
+name: E2E TEST - hosted inference - PRODUCTION
+
+on:
+  workflow_dispatch:
+    inputs:
+      skip_warmup:
+        type: boolean
+        description: "Do you want to skip services warmup?"
+        default: false
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🛎️ Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - name: 🐍 Set up Python 3.10
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+      - name: 🦾 Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements/requirements.test.unit.txt -r requirements/requirements.test.integration.txt -r requirements/requirements.code_analysis.txt
```

Comment:
We don't seem to be running code analysis in this action, do we need to install it?